xbmc/video/VideoInfoTag.cpp

```diff
@@ -556,16 +556,13 @@ void CVideoInfoTag::Serialize(CVariant& value) const
   value["episode"] = m_iEpisode;
   value["uniqueid"]["unknown"] = m_strUniqueId;
   value["rating"] = GetRating().rating;
-  value["ratings"] = CVariant(CVariant::VariantTypeArray);
   for (const auto& i : m_ratings)
   {
-    CVariant rating;
```

Comment:
I would change this a bit to reduce having to type all the indices multiple times to something like
```
CVariant ratings = CVariant(CVariant::VariantTypeObject);
for (const auto& i : m_ratings)
{
  CVariant rating;
  rating["rating"] = i.second.rating;
  rating["votes"] = i.second.votes;
  rating["default"] = i.first == m_strDefaultRating;

  ratings[i.first] = rating;
}
value["ratings"] = ratings;
```
This way you only have to use every index once.